src/app.fetch-httpClient.config.js

```diff
@@ -45,6 +53,40 @@ export class FetchConfig {
         request.headers.append(config.authHeader, token);
 
         return request;
+      },
+      response(response, request) {
+        return new Promise(
+          (resolve, reject) => {
+            if (response.ok) {
+              resolve(response);
+            }
+            if (response.status === 401) {
+              if (auth.isTokenExpired() && config.httpInterceptor) {
+                let refreshTokenName: string = config.refreshTokenPrefix ? `${config.refreshTokenPrefix}_${config.refreshTokenName}` : config.refreshTokenName;
+                if (storage.get(refreshTokenName)) {
+                  auth.updateToken()
+                    .then(
+                    () => {
+                      let token = storage.get(that.tokenName);
+                      if (config.authHeader && config.authToken) {
+                        token = `${config.authToken} ${token}`;
+                      }
+                      request.headers.append('Authorization', token);
+                      client.fetch(request)
+                        .then(resp => {
+                          resolve(resp);
+                        });
+                    }
+                    ).catch(err => {
+                      reject(err);
+                    });
```

Comment:
shorter
`` = delete it

----
catch(err => {reject(err);});
is
catch(reject);
is
nothing since reject is called on error already